src/store/modules/products.js

```diff
@@ -81,9 +83,17 @@ export default {
       const { productId, recipientId } = data;
       await transferProduct(productId, recipientId);
     },
-    async postPicture({commit}, {productId, image}) {
-      await postPicture(productId, image)
-    }
+    async postPicture({ commit }, { productId, image }) {
+      await postPicture(productId, image);
+    },
+    async getPicutre({ commit }, productId) {
+      try {
+        const res = await fetchPicture(productId);
+        return URL.createObjectURL(new Blob([res], { type: "image/png" }));
+      } catch (error) {
+        return null
+      }
```

Comment:
Should we return/throw the error to show the user what went wrong?

Reply:
i did it that way so we dont break the app with the error since I cannot get snackbar here, but will try with throw error

Reply:
fixed